### PR TITLE
Employees Page Front-End

### LINKED ISSUE #119: Task: Implement the front-end for the "My employees" page
The my employees section will list all the employees associated to the company they work for. This will display employee cars with their details. This page will be the page where they can edit their role and building(s).

### PR DESCRIPTION
**Pull request by npiperni**
closes #119

### CODE REVIEW

**Review by Tharsipa — APPROVED** [review]

**Review by CarlaShawi — APPROVED** [review]
looks good

**Review by amczuboka — CHANGES_REQUESTED** [review]
Overall looks great, could you change the colour of the text when the option is hovered to see it better. 
Screenshot 2024-03-01 at 11 31 23 AM

**Review by Ziadsharkos — APPROVED** [review]

**Comment by npiperni** [comment]
> Overall looks great, could you change the colour of the text when the option is hovered to see it better. Screenshot 2024-03-01 at 11 31 23 AM

Fixed

File: src/app/components/employees-list/employees-list.component.ts [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+  isAllChecked() {
+    return this.employees.every(_ => _.checked);
+  }
+
+  isSomeItemsChecked() {
+    return this.employees.some(_ => _.checked);
+  }
+
+  remeveSelectedItems() {
```
Comment (Guillaume-Lachapelle) [inline src/app/components/employees-list/employees-list.component.ts:48]: There is a typo in the name. Rename to removeSelectedItems()

**Review by Guillaume-Lachapelle — APPROVED** [review]

**Review by KarinaSandur — APPROVED** [review]
Looks good!

**Review by IsabelleCzuboka — APPROVED** [review]
Looks good

**Review by amczuboka — APPROVED** [review]
Nice UI.

**Pull request merged**